doc.md

```diff
@@ -65,6 +65,28 @@ to consult the documentation for the release you are using.
       [1.1](tabix-1.1.html),
       [1.0](tabix-1.0.html))
 
+## HowTos
+
+[BCFtools](http://samtools.github.io/bcftools/howtos)
+
+- Installation
+    - [Latest development version](http://samtools.github.io/bcftools/)
```

Comment:
Do you really mean http://samtools.github.io/bcftools/ and not http://samtools.github.io/bcftools/howtos/install.html?  Both exist, but differ.

I'm also wondering if we should link to https://github.com/samtools/bcftools/blob/develop/INSTALL or at least update these with that information.  Eg discussion on bzip2 and lzma dependencies (and arguably we could add how to disable these if you don't care about cram support in your bcftools binary).

Reply:
You are right updated now.